src/main/java/frc/robot/RobotContainer.java

```diff
@@ -272,10 +273,23 @@ private void configureMechBindings() {
         }, tiltedElevatorSubsystem));
 
         signalLEDSubsystem.setDefaultCommand(new RunCommand(() -> {
-            if (blSwitch.getAsBoolean()) {
-                signalLEDSubsystem.setColor(255, 100, 0);
+            double x = mechController.getRightX();
+            double y = mechController.getRightY();
+
+            if(mechController.getRightStickButtonPressed()){
```

Comment:
desired functionality is stick up = yellow, stick down = purple. if the stick is pressed down, color wheel